iis-wf/iis-wf-transformers/pom.xml

```diff
@@ -1,72 +1,68 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-	
-	<modelVersion>4.0.0</modelVersion>
-    
-	<parent>
-		<groupId>eu.dnetlib.iis</groupId>
-		<artifactId>iis-wf</artifactId>
-		<version>1.2.0-SNAPSHOT</version>
-	</parent>
-	
-	<artifactId>iis-wf-transformers</artifactId>
-	<packaging>jar</packaging>
-	
-		
-	<dependencies>
-	     <dependency>
-	        <groupId>org.apache.pig</groupId>
-	        <artifactId>piggybank</artifactId>
-	     </dependency>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 
-		 <dependency>
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>eu.dnetlib.iis</groupId>
+        <artifactId>iis-wf</artifactId>
+        <version>1.2.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>iis-wf-transformers</artifactId>
+    <packaging>jar</packaging>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.pig</groupId>
+            <artifactId>piggybank</artifactId>
+        </dependency>
+
+        <dependency>
             <groupId>eu.dnetlib.iis</groupId>
             <artifactId>iis-schemas</artifactId>
             <version>${project.version}</version>
         </dependency>
-        
+
         <dependency>
             <groupId>eu.dnetlib.iis</groupId>
             <artifactId>iis-common</artifactId>
             <version>${project.version}</version>
         </dependency>
-        
+
         <dependency>
             <groupId>eu.dnetlib.iis</groupId>
             <artifactId>iis-common</artifactId>
             <version>${project.version}</version>
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
-        
+
+        <dependency>
+            <groupId>com.beust</groupId>
+            <artifactId>jcommander</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-common</artifactId>
```

Comment:
`hadoop-common` was `provided` before. Do we need to provide this dependency in a package right now?

Reply:
This is a consequence of a change in spark dependencies scope in `spark-utils` - with new `1.0.1` release spark dependencies are scoped as `provided`. Previously they were scoped as `compile` and this meant that whenever `spark-utils` was used it brought all spark dependencies with itself, together with `hadoop-common`. These dependencies were put on classpath and were also available in the module using spark-utils, as if the module depend on them directly. After changing the scope of spark libs in spark-utils, `hadoop-common` had to be explicitly linked in `iis-wf-transformer`.